pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -51,15 +52,16 @@ class ModificationsExportDialog extends React.Component {
     }
 
     handleSelect(event, active_tab) {
-        this.setState({ active_tab });
+        this.setState({ active_tab, copied: false });
+        clearTimeout(this.timeoutId);
```

Comment:
We usually call this as `window.clearTimeout`, and it should explicitly check if `timeoutId  !== null`. It *seems* to work with calling with null in current Firefox, but I wouldn't bet on it on all browsers.